extract.py

```diff
@@ -1,31 +1,72 @@
 from pathlib import Path
 import shutil
 from tarfile import TarFile
+from typing import Union
+
+import logging
+logging.basicConfig(level=logging.INFO)
 
 
 def main():
+    # Find the RAW tar files and recursively extract them
     projects_dir = Path.cwd() / 'projects'
     for project_dir in projects_dir.iterdir():
         if project_dir.is_dir() and not project_dir.is_symlink():
             geo_dir = project_dir / 'geo'
             for geo_file in geo_dir.iterdir():
                 if geo_file.name.endswith('_RAW.tar'):
```

Comment:
Didn't we say we want to recursively extract all tars.

Reply:
Ah ok, I thought we only wanted to start with the '_RAW' tar files then recursively act on the extracted files.